Tests/NIOTransportServicesTests/NIOTSBootstrapTests.swift

```diff
@@ -371,6 +371,60 @@ final class NIOTSBootstrapTests: XCTestCase {
         XCTAssertEqual(try listenerChannel.getOption(NIOTSChannelOptions.multipathServiceType).wait(), .handover)
         XCTAssertEqual(try connectionChannel.getOption(NIOTSChannelOptions.multipathServiceType).wait(), .handover)
     }
+
+    func testNWParametersConfigurator() async throws {
+        final class WaitForConnectionHandler: ChannelInboundHandler, Sendable {
+            typealias InboundIn = Never
+
+            let connectionPromise: EventLoopPromise<Void>
+
+            init(connectionPromise: EventLoopPromise<Void>) {
+                self.connectionPromise = connectionPromise
+            }
+
+            func channelActive(context: ChannelHandlerContext) {
+                self.connectionPromise.succeed()
```

Comment:
It won't matter in this test, but generally speaking handlers should forward handler lifecycle events. If they don't, any other handlers in the pipeline waiting for the event might not work as expected.